docs/installation/autoware/source-installation.md

```diff
@@ -20,7 +20,9 @@ sudo apt-get -y update
 sudo apt-get -y install git
 ```
 
-> Note: If you wish to use ROS 2 Galactic on Ubuntu 20.04, refer to installation instruction from [galactic](https://autowarefoundation.github.io/autoware-documentation/galactic/installation/autoware/source-installation/) branch, but be aware that Galactic version of Autoware might not have latest features.
+!!! note
+
+    If you wish to use ROS 2 Galactic on Ubuntu 20.04, refer to installation instruction from [galactic](https://autowarefoundation.github.io/autoware-documentation/galactic/installation/autoware/source-installation/) branch, but be aware that Galactic version of Autoware might not have latest features.
```

Comment:
I think these refactor-like changes should be done in another pull request.

Reply:
okay. I will revert this change. :+1:

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/369/commits/0708589875b532b8db3af2e7f45fcc474f7410d7